website/static/js/pages/shareEmbedConfig.js

```diff
@@ -0,0 +1,7 @@
+var pym = require('pym.js');
+
+var pymParent = new pym.Parent(
+    'share_registration_iframe',
+    'http://localhost:8000/provider_registration/',
```

Comment:
This will definitely need to be updated with whereever the external django app ends up being hosted